openmls/src/group/core_group/mod.rs

```diff
@@ -909,7 +909,7 @@ impl CoreGroup {
         };
 
         // Build AuthenticatedContent
-        let mut commit = AuthenticatedContent::commit(
+        let mut authenticated_content = AuthenticatedContent::commit(
```

Comment:
Just double-checking: is this renaming intended or some leftover from earlier?

Reply:
It's part of me debugging issues. And I found the name here confusing.
I'll click the auto-merge button. Please request change if you want it changed back.